deepchecks/nlp/utils/embeddings_display.py

```diff
@@ -0,0 +1,163 @@
+        print(f'Avg outlier score is {np.mean(prob_vector)}')
+        if indexes_to_highlight is not None:
+            print(f'Avg outlier score in real outlier samples is {np.mean(prob_vector[text_dataframe.index.isin(indexes_to_highlight["real outlier samples"])])}')
+
+    # save to file
+    text_dataframe.to_csv(os.path.join(path, "outlier_metadata.tsv"), sep="\t")
+
+    # save_embeddings_to_file
+    checkpoint = tf.train.Checkpoint(outlier_embedding=tf.Variable(embeddings_df))
+    checkpoint.save(os.path.join(path, "outlier_embedding.ckpt"))
+
+def _select_highlight_value(index, indexes_to_highlight: Dict[str, List[int]]):
+    for key, value in indexes_to_highlight.items():
+        if index in value:
+            return key
+    return "other"
+
+def create_drift_display(train_text: pd.Series, test_text: pd.Series,
+                         train_embeddings: np.ndarray, test_embeddings: np.ndarray,
+                         path: str, sample_size: int = 2500, verbose: bool = False,
+                         indexes_to_highlight: Dict[str, List[int]] = None):
+    if not os.path.exists(path):
+        os.makedirs(path)
+
+    train_dataframe, test_dataframe = pd.DataFrame(train_text), pd.DataFrame(test_text)
+    train_dataframe.columns, test_dataframe.columns = ['text'], ['text']
+    train_embeddings = pd.DataFrame(train_embeddings, index = train_dataframe.index)
+    test_embeddings = pd.DataFrame(test_embeddings, index = test_dataframe.index)
+
+    if len(train_dataframe) > sample_size:
+        train_dataframe = train_dataframe.sample(sample_size)
+        train_embeddings = train_embeddings.loc[train_dataframe.index]
+    if len(test_dataframe) > sample_size:
+        test_dataframe = test_dataframe.sample(sample_size)
+        test_embeddings = test_embeddings.loc[test_dataframe.index]
+
+    all_embeddings = pd.concat([train_embeddings, test_embeddings], ignore_index=True)
+    all_embeddings_labels = [0] * len(train_embeddings) + [1] * len(test_embeddings)
+    x_train, x_test, y_train, y_test = train_test_split(floatify_dataframe(all_embeddings), all_embeddings_labels,
+                                                        stratify=all_embeddings_labels, test_size=0.2)
+
+    # train a model to disguise between train and test samples
+    domain_classifier = GradientBoostingClassifier(max_depth=3, subsample=0.8, min_samples_split=50, n_estimators=30)
+    domain_classifier.fit(x_train, y_train)
+
+
+    domain_classifier_auc = roc_auc_score(y_test, domain_classifier.predict_proba(x_test)[:, 1])
+    drift_score = auc_to_drift_score(domain_classifier_auc)
+    if verbose:
+        print('domain_classifier_auc on train is ', roc_auc_score(y_train, domain_classifier.predict_proba(x_train)[:, 1]))
+        print('domain_classifier_auc on test is ', domain_classifier_auc)
+        print(f'Domain classifier drift score is {drift_score:.2f}')
```

Comment:
What about adding the drift bar here?
Should be easy, I think - just take the Figure from plot.py and do .show() and should work in ipynb

Reply:
Its added when verbose = False